thrift/shared.thrift

```diff
@@ -1637,3 +1637,15 @@ struct GetWorkflowExecutionRawHistoryResponse {
   20: optional binary nextPageToken
 }
 
+struct LongPollWorkflowExecutionRawHistoryRequest {
+  10: optional string domain
+  20: optional WorkflowExecution execution
+  30: optional i32 maximumPageSize
+  40: optional binary nextPageToken
+  50: optional HistoryEventFilterType HistoryEventFilterType
+}
+
+struct LongPollWorkflowExecutionRawHistoryResponse {
```

Comment:
Although they are the same, I think it might be a little bit unclear if the request is 
PollForWorkflowExecutionRawHistoryRequest and response is GetWorkflowExecutionRawHistoryResponse.

Reply:
Makes sense.